probedata.go

```diff
@@ -102,6 +103,31 @@ type StreamDisposition struct {
 	AttachedPic     int `json:"attached_pic"`
 }
 
+// Chapters is a json data structure to represent chapters.
+type Chapter struct {
+	ID               int     `json:"id"`
+	TimeBase         string  `json:"time_base"`
+	StartTimeSeconds float64 `json:"start_time,string"`
+	EndTimeSeconds   float64 `json:"end_time,string"`
+	TagList          Tags    `json:"tags"`
+}
+
+// StartTime returns the start time of the chapter as a time.Duration
+func (c *Chapter) StartTime() time.Duration {
+	return time.Duration(c.StartTimeSeconds * float64(time.Second))
+}
+
+// EndTime returns the end timestamp of the chapter as a time.Duration
+func (c *Chapter) EndTime() time.Duration {
+	return time.Duration(c.EndTimeSeconds * float64(time.Second))
+}
+
+// Name returns the value of the "title" tag of the chapter
+func (c *Chapter) Name() string {
```

Comment:
Why name the function `Name` when it returns the title? Isnt it better to name it `Title` then?